src/main/kotlin/com/kotlindiscord/database/migrator/commands/MigrateAll.kt

```diff
@@ -0,0 +1,21 @@
+package com.kotlindiscord.database.migrator.commands
+
+import com.kotlindiscord.database.connectToDb
+import com.kotlindiscord.database.migrator.MigrationsManager
+import picocli.CommandLine.Command
+import java.util.concurrent.Callable
+
+@Command(
+    name = "all",
+    description = ["Apply all unapplied migrations."]
+)
+class MigrateAll : Callable<Int> {
+
```

Comment:
This class has a big ol' forehead

Reply:
Pretty sure I've fixed these by moving everything onto a single line.

Reply:
It's the way blank lines are being used, not really the density